build_defs/build_defs.bzl

```diff
@@ -17,14 +18,14 @@ def _run_schema2pkg(name, src, out, language_name, language_flag, file_extension
         name = name,
         srcs = [src],
         outs = [out],
-        deps = ["//src/tools:schema2pkg_srcs"],
+        deps = ["@arcs//src/tools:schema2pkg_srcs", "@arcs//tools"],
```

Comment:
How come you've added the `@arcs` prefix here? Shouldn't these targets work happily without that?

Reply:
There's some weird stuff going on with multi-workspace builds I'm trying to sort out. 

Because we're using build macros, not rules, when you import a build macro from the Arcs workspace into the arcs_javaharness workspace,  it's deps will be stuff like '//src' or '//build_defs', but inside of the javaharness workspace, those don't exist. 

This is the problem with using macros, they don't know about workspaces, they're just python functions. 

This is causing so much trouble, I'm thinking about eliminating the multi-workspace, and changing all BUILD files to be based off of the root workspace.